vision/google/cloud/vision/annotations.py

```diff
@@ -89,8 +89,11 @@ def from_api_repr(cls, response):
         annotations = {}
         for feature_type, annotation in response.items():
             curr_feature = annotations.setdefault(_KEY_MAP[feature_type], [])
-            curr_feature.extend(
-                _entity_from_response_type(feature_type, annotation))
+            entity = _entity_from_response_type(feature_type, annotation)
+            if isinstance(entity, list):
+                curr_feature.extend(entity)
+            else:
+                annotations[_KEY_MAP[feature_type]] = entity
```

Comment:
Yeah, I'm thinking of handling it more like how I'm doing the pb versions with the helper methods that do the conversions.

WDYT?